app/scripts/move-popup/dimMoveItemProperties.directive.js

```diff
@@ -186,5 +154,69 @@
         });
       }
     }
+
+    /*
+     * Determins if the item has damange and it's non kinetic.
+     *
+     * @param {Object} item - A item from the destiny server
+     * @return {Boolean}
+     */
+    function itemHasNonKineticDamage(item) {
+      if (!item.dmg || item.dmg === 'kinetic') {
+        return false;
+      }
+      return true;
+    }
+
+    /*
+     * Check for a lockable or elemental property in an item.
+     *
+     * @param {Object} item - A item from the destiny server
+     * @return {Boolean}
```

Comment:
To expand on my point above, it's a question of judgement. If I see `itemHasNonKineticDamage(item)`, I'm pretty clear about what the function does, what it returns, and what the `item` parameter is. If I see `computeQuality(item, stat)` I might know what `item` is, but not `stat` which is less common, and the overall contract for `computeQuality` and what it returns, what its goal is, etc., could use docs. Or if there's an odd behavior or side effect. 

Likewise describing terms and concepts are often valuable, and best done where that concept is centrally defined.